### PR TITLE
Add text on security bugs.

### PR DESCRIPTION
**Pull request by phoglund**

### CODE REVIEW

File: bugs/index.md [hunk 1]
```diff
@@ -27,6 +27,15 @@ triaged by Chrome and WebRTC engineers.
 Please include as many relevant details as possible.
 
 
+### Filing a Security Bug
+
+The WebRTC team takes security very seriously. If you find a vulnerability in
+WebRTC, please file a [Chromium security bug][ChromeSecurity], even if the bug
+only affects native WebRTC code and not Chromium.
+
+Any vulnerabilities will be disclosed on webrtc-discuss once we have fixed the
```
Comment (ErikHellman) [inline bugs/index.md]: discuss-webrtc?

Maybe a link to the group as well? https://groups.google.com/group/discuss-webrtc

**Review by ErikHellman — CHANGES_REQUESTED** [review]
Two minor issues.

File: bugs/index.md (hunk 1 above)
Comment (phoglund) [reply bugs/index.md]: Good catch, thanks

**Comment by phoglund** [comment]
PTAL

**Review by ErikHellman — APPROVED** [review]
LGTM

**Pull request merged**